components/Shell.tsx

```diff
@@ -36,14 +36,18 @@ export default function Shell(props) {
 
   return session ? (
     <div>
-      <div className="bg-gradient-to-b from-blue-600 via-blue-600 to-blue-300 pb-32">
-        <nav className="bg-blue-600">
+      <div className="dark:from-gray-900 dark:to-gray-900  bg-gradient-to-b from-blue-600 via-blue-600 to-blue-300 pb-32">
+        <nav className="dark:bg-gray-900 bg-blue-600">
           <div className="max-w-7xl mx-auto sm:px-6 lg:px-8">
-            <div className="border-b border-blue-500">
+            <div className="border-b border-blue-500 dark:border-gray-">
```

Comment:
Looks like you're missing the last part of the colour code (e.g. border-gray-**400**)

Reply:
wtf. fixing .. thanks!